Docs/source/building/summit.rst

```diff
@@ -3,87 +3,107 @@
 Building WarpX on Summit (OLCF)
 ================================
 
-For the `Summit cluster
-<https://www.olcf.ornl.gov/summit/>`__ at OLCF,
-use the following commands to download the source code, and switch to the
-correct branch:
+The `Summit cluster <https://www.olcf.ornl.gov/summit/>`_ is located OLCF.
 
-::
+If you are new to this system, please see the following resources:
 
-    mkdir warpx_directory
-    cd warpx_directory
+* `Summit user guide <https://docs.olcf.ornl.gov/systems/summit_user_guide.html>`_
+* Batch system: `LSF <https://docs.olcf.ornl.gov/systems/summit_user_guide.html#running-jobs>`_
+* `Production directories <https://docs.olcf.ornl.gov/data/storage_overview.html>`_:
 
-    git clone https://github.com/ECP-WarpX/WarpX.git
-    git clone https://bitbucket.org/berkeleylab/picsar.git
-    git clone --branch development https://github.com/AMReX-Codes/amrex.git
+  * ``$PROJWORK/$proj/``: shared with all members of a project (recommended)
+  * ``$MEMBERWORK/$proj/``: single user (usually smaller quota)
+  * ``$WORLDWORK/$proj/``: shared with all users
+  * Note that the ``$HOME`` directory is mounted as read-only on compute nodes.
+    That means you cannot run in your ``$HOME``.
 
-Then, ``cd`` into the directory ``WarpX`` and use the following set of commands to compile:
 
-::
+Installation
+------------
 
-    module load gcc
-    module load cuda
-    make -j 16 COMP=gcc USE_GPU=TRUE
+We use the following modules and environments on the system.
 
-See :doc:`../running_cpp/platforms` for more information on how to run WarpX on Summit.
+.. code-block:: bash
 
-See :doc:`../visualization/yt` for more information on how to visualize the simulation results.
+   # please set your project account
+   export proj=<yourProject>
+
+   # required dependencies
+   module load gcc/6.4.0
```

Comment:
Would we want to specify gcc version. or just default module load gcc?

Reply:
It's also the default. I just put it because keeping the compiler fixed it more stable between module default updates in case one builds own software (as below) on top of it.